libc/docs/signal.rst

```diff
@@ -0,0 +1,41 @@
+signal.h Functions
```

Comment:
You're going to need an include akin to e127997155a1cd1c5692c42aad074064c8dad099 else we'll break the build again.

Run `ninja docs-libc-html` to build the docs and verify the build isn't broken.

You'll probably need to configure llvm-libc with `-DLLVM_ENABLE_SPHINX=ON -DLIBC_INCLUDE_DOCS=ON` cmake options for that `ninja` target to be recognized.